rules/windows/defense_evasion_disable_windows_firewall_rules_with_netsh.toml

```
@@ -14,6 +14,39 @@ index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
 language = "eql"
 license = "Elastic License v2"
 name = "Disable Windows Firewall Rules via Netsh"
+note = """## Triage and analysis
+
+### Investigating Disable Windows Firewall Rules via Netsh
+
+The Windows Defender Firewall is native component which provides host-based, two-way network traffic filtering for a
+device, and blocks unauthorized network traffic flowing into or out of the local device.
+
+Attackers can disable firewall rules which are intended to prevent lateral movement and command and control traffic to
+enable their operations.
+
+This rule identifies patterns related to the disable of firewall rules using the `netsh.exe` utility.
+
+#### Possible investigation steps:
+
+- Identify the user account which performed the action and whether it should perform this kind of action.
+- Contact the user to check if it is aware of the operation.
+- Investigate script execution chain (parent process tree).
+- Investigate other alerts related to the user/host in the last 48 hours.
+- Analyze the command run to determine what was allowed by it.
+
+### False Positive Analysis
+
+- This mechanism can be used legitimately. Check whether the user should be performing this kind of activity, is aware
```

```suggestion
- This `netsh.exe` command line tool can be used legitimately. Check whether the user should be performing this kind of activity, is aware
```

See @terrancedejesus update as well.